cypress/integration/error_spec.js

```diff
@@ -1,8 +1,48 @@
 describe('Error Component', () => {
-  it('should display the nav bar and message', () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        method: 'POST',
+        url: 'https://not-fur-gotten-be.herokuapp.com/graphql',
+        headers: {
+          'x-gql-operation-name': 'getAllPets',
+        },
+        times: 1
+      },
+      {
+        "errors": [
```

Comment:
This was a wonderful find for figuring out how to simulate an error in this GRAPHQL post request. Yay for the docs!